unfoldingWord/gateway-edit
# Feature 228/add support for more urls

## Describe what your pull request addresses
part of https://github.com/unfoldingword/gateway-edit/issues/228
- [ ] added support for more URL formats

## Test Instructions
- test with https://deploy-preview-265--gateway-edit.netlify.app/
- try adding the following scripture urls to a scripture pane.  Make sure scripture actually shows up in scripture pane:
- [ ] `https://git.door43.org/unfoldingWord/en_ult`
- [ ] `https://git.door43.org/unfoldingWord/el-x-koine_ugnt/src/tag/v0.20`
- [ ] `https://door43.org/u/unfoldingWord/en_ult/`
- [ ] `https://git.door43.org/Door43-Catalog/en_ust.git`
- [ ] `git@git.door43.org:unfoldingWord/en_ult.git`
- [ ] change bible reference to heb
- [ ] change a scripture pane to `https://git.door43.org/ru_gl/ru_rsob` and should see message that `This book can not be found in the project...`.  Should not get a crash
- [ ] switch book to tit and should see Russian content.

# Reviewable: https://reviewable.io/reviews/unfoldingWord/gateway-edit/265#-
